imsv-docs-astro/src/content/docs/guides/funding-protocols/universal-evm-smart-contract.mdoc

```diff
@@ -6,33 +6,43 @@ fundingProtocol:
   source: https://github.com/immersve/funding-contract-universal-evm
 ---
 
-The Immersve Universal EVM protocol allows client applications to fund Immersve
-cards via simple ERC-20 transfers. Depositors can withdraw funds at any time
-with signed withdrawal approvals. Depositor funds can only be settled to
-Immersve’s approved settlement address. Deposits are scoped to the client
-application that registers the Funds Storage contract as an Immersve Funding
-Channel.
+The Immersve Universal EVM [funding protocol](/guides/funding-protocols) allows
+client applications to fund Immersve cards via simple ERC-20 transfers.
+Depositors can withdraw funds at any time with signed withdrawal approvals.
+Depositor funds can only be settled to Immersve’s approved settlement address.
+Deposits are scoped to the client application that registers the Funds Storage
+contract as an Immersve Funding Channel.
 
-## Components
+## Smart Contract
 
-![polygonscan-contract-landing](@assets/universal-evm/overview-2.png)
+The Universal EVM protocol has a master "Funds Manager" contract and many
+partner "Funds Storage" contracts.
+
+
+### Source
+{% link href="https://github.com/immersve/funding-contract-universal-evm" /%}.
+
+### Deployed Contract Addresses
+
+The Funds Manager contract is designed to work on any EVM. Funds Manager Contract addresses
+can be found at [Supported Chains](/guides/supported-chains/)
 
-### Funds Manager Contract
 
-The Funds Manager contract is the main entry point and is a factory for Funds
-Storage contracts. It is also used by Immersve for performing administrative
-operations. The `createFundsStorage` function can be used by any message signer
-to deploy a Funds Storage contract address.
+### Smart Contract Overview
 
-### Funds Storage Contract
+The Funds Manager contract is the main entry point and is a factory for the
+Funds Storage contracts. Funds Storage contracts are the target for depositor
+ERC-20 transfers that are scoped to an integrated partner application. The
+Funds Manager contract is also used by Immersve for performing settlements and
+refunds.
 
-Funds Storage contracts are the target for depositor ERC-20 transfers. The
-Funds Storage contract is scoped to the integrating application. It only needs
-to be deployed once for all depositors. After it has been deployed it must be
-registered with the Immersve backend using the [Create Funding Channel
-API](https://docs.immersve.com/api-reference/create-a-funding-channel).
 
-## Deposits
+![polygonscan-contract-landing](@assets/universal-evm/overview-2.png)
+
+
+## Protocol Mechanics
+
+### Deposits
```

Comment:
Non-blocking
The detail and diagrams on mechainics from the Card Funding should eventually be moved here. Funding Source is introduced but not given a lot of context. That context should be the responsibility of the Card Funding guide.

Great job!

Reply:
💯